test/user.test.js

```diff
@@ -214,6 +214,31 @@ describe('User', function() {
       assert(u2.password === u1.password);
     });
 
+    it('invalidates the user\'s accessToken when the user is deleted', function(done) {
+      User.create({ email: 'b@c.com', password: 'bar' }, function(err, user) {
```

Comment:
I prefer the cb, as it looks organized in my opinion, as functions are defined and could be easily accessed for modification and testing.